source/pervasive/multiset.rs

```diff
@@ -210,7 +223,7 @@ pub proof fn axiom_multiset_contained<V>(m: Map<V, nat>, v: V)
         m.dom().finite(),
         m.dom().contains(v),
     ensures 
-        #[trigger] Multiset::new(m).count(v) == m[v],
+        #[trigger] Multiset::from_map(m).count(v) == m[v],
```

Comment:
On line 215 please replace
```
// Specifications of `new`
```

with
```
// Specifications of `from_map`
```

Reply:
@utaal I replaced the TODO with a comment on the axiom-of-concern, warning anyone who might try to enhance the library in the future.